<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalPreferences</class>
 <widget class="QDialog" name="GlobalPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" rowspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Viewing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="updateDisplayGroupBox">
        <property name="title">
         <string>Update display on:</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QRadioButton" name="sliderMoveRadioButton">
           <property name="toolTip">
            <string>Choosing this will update the display as sliders are being moved</string>
           </property>
           <property name="text">
            <string>Slider move</string>
           </property>
           <attribute name="buttonGroup">
            <string>updateDisplayButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="sliderReleaseRadioButton">
           <property name="toolTip">
            <string>Choosing this will improve performance on slower computers</string>
           </property>
           <property name="text">
            <string>Slider release</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>updateDisplayButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="startUpRefreshCheck">
        <property name="toolTip">
         <string>Check this if screen doesn't refresh on startup</string>
        </property>
        <property name="text">
         <string>Refresh screen on startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="boxModelCheck">
        <property name="toolTip">
         <string>Enabling this will improve rendering performance</string>
        </property>
        <property name="text">
         <string>Box model with mouse move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawBoxAlwaysCheck">
        <property name="text">
         <string>Bounding Box always</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="directRenderingCheck">
        <property name="toolTip">
         <string>Fix X Server memory problem on Solaris</string>
        </property>
        <property name="text">
         <string>Direct Rendering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawAxisCheck">
        <property name="text">
         <string>Draw Axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="drawAxisOriginCheck">
        <property name="text">
         <string>Draw Axis Origin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Notes:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Direct Rendering:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Fixes X Sun memory problem&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Box Model:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Replaces object with a box when mouse interacts with the screen. Renders mouse movement faster.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>startUpRefreshCheck</zorder>
     <zorder>boxModelCheck</zorder>
     <zorder>directRenderingCheck</zorder>
     <zorder>drawAxisCheck</zorder>
     <zorder>textEdit</zorder>
     <zorder>drawAxisOriginCheck</zorder>
     <zorder>updateDisplayGroupBox</zorder>
     <zorder>drawBoxAlwaysCheck</zorder>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>11</number>
     </property>
     <property name="margin">
      <number>11</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="toolTip">
        <string>Commit to changes made to global preferences</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Cancels all changes made to global preferences</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Default image export format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QComboBox" name="imageFormatComboBox">
        <property name="toolTip">
         <string>Click to select the default image export format from a list</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Mouse buttons</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QSpinBox" name="mouseButtonsSpinBox">
        <property name="toolTip">
         <string>Specifies how many buttons are  on the mouse. Read manual on keyboard mapping with two/three button mouse modes</string>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Bin Palettes</string>
     </property>
     <property name="buddy">
      <cstring>binPalettesSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="binPalettesSpinBox"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>11</number>
      </property>
      <item>
       <widget class="QCheckBox" name="restoreWSCheck">
        <property name="toolTip">
         <string>Restores the last workspace window size</string>
        </property>
        <property name="text">
         <string>Restore window size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="openScriptFileCheck">
        <property name="toolTip">
         <string>The current scriptfile being used will be executed next time</string>
        </property>
        <property name="text">
         <string>Always execute current scriptfile:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scriptFileNameLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showSplashScreenCheck">
        <property name="toolTip">
         <string>Click to show or hide the splashscreen</string>
        </property>
        <property name="text">
         <string>Show splashscreen on startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Changes the path for where to look for program documentation</string>
        </property>
        <property name="text">
         <string>Documentation:</string>
        </property>
        <property name="buddy">
         <cstring>docPathLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="docPathLineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GlobalPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>GlobalPreferences</receiver>
   <slot>slotOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOk()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="updateDisplayButtonGroup"/>
 </buttongroups>
</ui>
